src/app/Home/components/onBoardingModal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const modal = ({ show, close }) => (
```

Comment:
It would be nice if we capitalize the first letter of components.

Reply:
@Chris-Maina  I have the above.